docker/Dockerfile

```diff
@@ -201,6 +199,7 @@ USER $MAMBA_USER
 
 EXPOSE 22
 
+
```

Comment:
Extra blank line? I think the rest of the file has a single blank line separating statements?